README.md

```diff
@@ -15,15 +20,17 @@ If you’re using this demo, please **★Star** this repository to show your int
 > [go/microservices-demo](http://go/microservices-demo) if you are using this
 > application.
 
+Looking for the old Hipster Shop frontend interface? Use the manifests in release [v0.1.4](https://github.com/GoogleCloudPlatform/microservices-demo/releases/v0.1.4).
```

Comment:
mind if we link to https://github.com/GoogleCloudPlatform/microservices-demo/tree/v0.1.4/kubernetes-manifests instead? I'm not sure which one's better though. Maybe visitors can figure out.

Reply:
added both links!